### PR TITLE
Eslint config and little changes on send request

### PR DESCRIPTION
**Pull request by RabeaWahab**
add eslint config airbnb standards as it is known enough, plus cleaning up some indentation and little changes on logic, looking forward in moving this to es6

### CODE REVIEW

File: lib/verge.js [hunk 1]
```diff
@@ -1,195 +1,196 @@
...
+    this.opts.passphrasecallback(command, args, function (err, passphrase, timeout) {
+      if (err) {
+        fn(err);
+      } else {
+        self.send('walletpassphrase', passphrase, timeout, retry);
+      }
+    });
+  },
+
+  set:function(k, v) {
```
Comment (ekryski) [inline lib/verge.js]: This should probably have a space to be consistent

File: lib/verge.js [hunk 2]
```diff
@@ -1,195 +1,196 @@
...
-            var opt = this.opts[k.toLowerCase()]
-          }
-          return  opt; //new Error('No such option "'+k+'" exists');
-        }
-    },
+    return this[func].apply(this, arguments);
+  },
 
-    errors: errors
+  auth:function(user, pass) {
```
Comment (ekryski) [inline lib/verge.js]: Should have a space here for consistency

**Comment by ekryski** [comment]
A couple super minor comments. Looks good! Thanks! :shipit:

**Comment by RabeaWahab** [comment]
@ekryski done and done :),  please review.

**Pull request closed without merging**